docs/proposed/http-storage-node-protocol.rst

```diff
@@ -279,7 +279,7 @@ Such an announcement will resemble this::
 
   {
       "anonymous-storage-FURL": "pb://...",          # The old key
-      "gbs-anonymous-storage-url": "pb://...#v=1"    # The new key
+      "anonymous-storage-NURLs": ["pb://...#v=1"]    # The new keys
```

Comment:
Reading this now I find "key" a bit confusing.  I think the original intent was to describe the mapping key.  Maybe "item" or "mapping entry" would make more sense.